build.gradle

```diff
@@ -50,7 +50,11 @@ ext {
             junit_dataprovider  : 'com.tngtech.java:junit-dataprovider:1.11.0',
             mockito             : 'org.mockito:mockito-core:2.27.0',
             assertj             : 'org.assertj:assertj-core:2.9.1',
-            assertj_guava       : 'org.assertj:assertj-guava:2.0.1'
+            assertj_guava       : 'org.assertj:assertj-guava:2.0.1',
+
+            springBeans         : 'org.springframework:spring-beans:5.2.2.RELEASE',
+            jakartaInject       : 'jakarta.inject:jakarta.inject-api:1.0',
```

Comment:
Is this where `javax.inject.Inject` is from? I think there is also a slim standard dependency `javax.inject:javax.inject:1` which we could maybe use?

Reply:
Even though maybe it's better to take everything from `jakarta`, since `@Resource` is not part of any standard dependency AFAIK.

Reply:
`jakarta.inject:jakarta.inject-api` contains `javax.inject.Inject`, and `jakarta.annotation:jakarta.annotation-api` contains `javax.annotation.Resource`.
I took Jakarta because they provide the latest version of these annotations.